config.json

```diff
@@ -186,7 +186,7 @@
         "name": "Rna Transcription",
         "uuid": "342974d6-9083-4754-a6c5-ed1e19e40ec5",
         "practices": [],
-        "prerequisites": [],
+        "prerequisites": ["strings", "arrays", "objects"],
```

Comment:
probably array-transformations

Do we throw error in this exercise? If yes: errors

Reply:
Nope errors are not needed for this exercise